### DISCUSSION TITLE
Check out my new plugin to support multiple groups

### DISCUSSION THREAD

**Discussion by onmax** [tailwindlabs/tailwindcss#8074] [Show and tell]
So today, I was making a component where there is a lot of interactions and I am using a lot `group`. The problem with `group` is that it does not allow you to have multiple nested `group`. You can read more about this problem in this [great article](https://linu.us/how-to-nest-groups-in-tailwindcss).

So, I wrote this new plugin [Tailwind Labeled Groups](https://github.com/onmax/tailwindcss-labeled-groups) that allows you to have multiple named groups!

I would love to hear your feedback!

**Comment by onmax** [comment]
Btw, you can also check out this reflection I shared on Twitter 🙈

https://twitter.com/_onmax/status/1512923476788752388

Let me know what you think!

**Comment by reinink** [comment]
Hey! Good news — we just added nested group and multiple peer support using variant modifiers in Tailwind CSS v3.2 🥳

You can read about it in our [announcement blog post](https://tailwindcss.com/blog/tailwindcss-v3-2#nested-group-and-multiple-peer-support-using-variant-modifiers), but for quick reference here’s what it looks like:

```html
<div class="group/sidebar ...">
  <!-- ... -->
  <div class="group/navitem ...">
    <a
      href="#"
      class="opacity-50 group-hover/sidebar:opacity-75 group-hover/navitem:bg-black/75"
    >
      <!-- ... -->
    </a>
  </div>
  <!-- ... -->
</div>
```

This lets you give each group a clear name that makes sense for that context on the fly, and Tailwind will generate the necessary CSS to make it work.

**Comment by Carlsia** [comment]
This article is truly astounding. Appreciative for sharing.A commitment of appreciation is all together for the association, keep on sharing such an information. [Allied Universal eHub Website
](https://www.allieduniversalehub.net/)
